configure.ac

```diff
@@ -513,12 +513,20 @@ AS_IF([test x$have_check = x], [
     AC_CHECK_HEADERS([check.h],,AC_MSG_ERROR([Could not find CHECK headers]))
 ])
 
+AC_ARG_ENABLE([linux-caps], [AS_HELP_STRING([--enable-linux-caps],
+                            [Allow SSSD programs to use libcap to drop privileges])],
```

Comment:
Let's:
(1) mention a default value  (" [default=yes]")
(2) replace 'to drop' with 'to manage own ...'

But that's cosmetic.